crates/host/src/wasmbus/mod.rs

```diff
@@ -3953,10 +3883,6 @@ impl Host {
             (Some("auction"), Some("provider"), None, None) => {
                 self.handle_auction_provider(message.payload).await
             }
-            (Some("cmd"), Some(host_id), Some("la"), None) => Arc::clone(&self)
```

Comment:
I think we should leave support in for this for at least one version. Consider a user who is interacting with a lattice where they've upgraded their host versions, but are using v0.23 of wash: wash will still be issuing `.la` requests to the control interface, but the host(s) won't be able to handle them

So I think this PR should remove all uses of start/launch from wash, and then after a new version of wash has been out for a while, we can remove support in the host

This follows a general safe pattern of deprecating and removing functionality from a server:
1. mark as deprecated in API
2. remove all uses from all clients
3. have reasonable knowledge all clients are updated
4. remove support from the server

I think we're jumping from 1) to 4) here

Reply:
As far as I understood here, this PR does remove use from all clients (I know we aren't using it in wadm and everything else is in this repo now) and they had been deprecated for a few versions now. However, I understand the concern with keeping support around for older wash versions. I'm honestly ok with cutting it at this point, but if we want to be extra careful, we can keep it for now. The other option is to remove and just return an error that says this endpoint has been removed

Reply:
I need to double check on this, but I'm fairly sure when I cut control interface 0.29 it actually proxies to `scale_actor` from within the control interface. So it's been removed from clients for quite a while https://docs.rs/wasmcloud-control-interface/0.30.0/src/wasmcloud_control_interface/lib.rs.html#309

Reply:
@thomastaylor312 my point is we cannot remove from all clients in the same release tuple (clients, server) as we drop support from the server

But it sounds like @brooksmtownsend covered this, and we _actually_ removed usage from all clients a version ago 👀 if that's the case, then 2) and 3) are already done, and I'm good with this PR moving to step 4) 👍